src/core/tools/common/providers.common.cpp

```diff
@@ -39,19 +39,20 @@
 #include "native_aio_provider.linux.h"
 #include "simple_task_queue.h"
 #include "network.sim.h"
-#include "simple_logger.h"
 #include "dsn_message_parser.h"
 #include "thrift_message_parser.h"
 #include "raw_message_parser.h"
+#include "../../core/simple_logger.h"
```

Comment:
don't use relative-path.
```suggestion
#include "core/simple_logger.h"
```